tasks/Fedora.yml

```diff
@@ -31,6 +31,7 @@
         (not use_distro_packages)
 
 - name: Installing the WineHQ packages for Fedora
+  become: true
```

Comment:
All of the tasks that are not using the `set_fact` module require root level access (copy, file, and dnf).

Reply:
You are right. It is now added.